storage/google/cloud/storage/bucket.py

```diff
@@ -1303,6 +1299,11 @@ def make_public(self, recursive=False, future=False, client=None):
                       ``NoneType``
         :param client: Optional. The client to use.  If not passed, falls back
                        to the ``client`` stored on the current bucket.
+
+        :raises ValueError:
+            If ``recursive`` is True, and the bucket contains more than 256
+            blobs.  This is to prevent extremely long runtime of this
+            method.
```

Comment:
Is there a way to go past 256 blobs?

Reply:
The user can iterate over the blobs directly, deleting them.  See the error message below at line 1324.

Reply:
Suggestion for the documentation:

```
If ``recursive`` is True, and the bucket contains more than 256
            blobs.  This is to prevent extremely long runtime of this
            method. Use :meth:`Bucket.list_blobs` and call :meth:`Blob.make_public`:
            for each blob.
```

Also error message suggestion:
```
message = (
                     'Refusing to make public recursively with more than '
                     '%d objects. If you want to make every object '
                     'in this bucket public, iterate through objects using'
                     'Bucket.list_blobs() and for each Blob call Blob.make_public()`
```

Reply:
505587f